.circleci/setup_storage.sh

```diff
@@ -38,6 +38,7 @@ source $BASH_ENV
 echo $(GREEN "Authenticating with GCP storage...")
 echo $STORAGE_TOKEN > storage-key.json
 gcloud auth activate-service-account --key-file=storage-key.json
+rm storage-key.json
```

Comment:
On my machine, `rm` throws up a user prompt while `rm -f` does not. Should this change?